isminet/models/base.py

```
@@ -1,19 +1,33 @@
-from pydantic import BaseModel, Field, ConfigDict, ValidationInfo, field_validator
+from typing import (
+    Generic,
+    List,
+    Optional,
+    TypeVar,
+    get_args,
+    get_origin,
+    Dict,
+    Any,
+    Union,
+)
```

Ruff (0.8.2)

`typing.List` is deprecated, use `list` instead

(UP035)